_locales/en/messages.json

```diff
@@ -1958,7 +1958,7 @@
 		"message": "Mobile Privacy Browser"
 	},
 	"hub_products_main_promo_description": {
-		"message": "Browse ad- and tracker-free on the go with our updated and enhanced mobile browser. Now equipped with a private search engine!"
+		"message": "Browse ad-and tracker free on the go with our updated and enhanced mobile browser. Now equipped with a private search engine!"
```

Comment:
This is the wrong string.  Please revert to the old string.
The dashes are incorrect.